### PR TITLE
Mentors - filtering

### LINKED ISSUE #122: Mentors Section Filtering
There have been some changes to the design after discussing with design + leadership more thoroughly on what we need for filtering

- [x] Add a new filter for Expertise and remove the position filter 
- [x] Remove the "all mentors" radio button
- [x] Change radio buttons to be checkboxes 
- [x] Add virtual checkbox 
- [ ] Implement the filtering by expertise, company, on shift, and virtual (you can see how the data was set up last year in airtable for shifts and things like that)
- [ ] The default should be that all of the mentors that are on shift right now are selected

Here is the updated design 
Screenshot 2023-01-13 at 3 12 49 PM

### PR DESCRIPTION
**Pull request by deanframe**
Adds mentors filtering functionality on livesite

Fixes #122 .

Changelist:

- Adds mentor section filtering defined in ticket
- Adds empty state for mentors (no mentors match selected filters)
- Grays out cybersecurity workshop on mainsite

Notes:

- still experiencing some cache bs with airtable, so going to check functionality with on shift filtering tomorrow.

Tested:

- locally firefox

### CODE REVIEW

File: live-site/components/mentors-section/MentorsSection.styles.tsx [hunk 1]
```diff
@@ -142,7 +146,15 @@ const StyledMentorsPaginationContainer = styled.div`
...
     max-width: 95%;
   }
-`
+`;
+
+const StyledNoMentorsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 8px;
```
Comment (jzhzhang66) [inline live-site/components/mentors-section/MentorsSection.styles.tsx]: use em SORRY

File: live-site/components/mentors-section/MentorsSection.tsx [hunk 2]
```diff
@@ -47,26 +62,68 @@ const MentorsSection: React.FC = () => {
     : mentors;
 
   useEffect(() => {
-    setMentors(
+    console.log(data);
```
Comment (jzhzhang66) [inline live-site/components/mentors-section/MentorsSection.tsx]: remove console log

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
THANKS FOR WORKING ON THIS DEAN<333

I'm sure this was working for u at some point but the filtering isn't doing anything for me - is this part of the airtable bs ur dealing with? hahha
https://user-images.githubusercontent.com/59738880/213877426-5dc7aecf-c273-4700-98de-fcf670b5f848.mov

File: live-site/components/mentors-section/MentorsSection.styles.tsx (hunk 1 above)
Comment (deanframe) [reply live-site/components/mentors-section/MentorsSection.styles.tsx]: lmfao i have muscle memory i need to remember to shake off

**Comment by deanframe** [comment]
@jzhzhang66 should be good now!

File: live-site/components/mentors-section/MentorsSection.styles.tsx [hunk 3]
```diff
@@ -86,43 +86,47 @@ const StyledMentorsListContainer = styled.div`
...
-`
+`;
 
 const StyledMentorContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
 
-  padding: 0.75em;
+  padding: 0.75em 0em;
```
Comment (jzhzhang66) [inline live-site/components/mentors-section/MentorsSection.styles.tsx]: can drop the em on the 0

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
HI DEAN!!! I'm still having some trouble with the filtering - it seems like the text on the filter doesn't change from "all" so it looks like it's not actually selecting any of the other options?
https://user-images.githubusercontent.com/59738880/213967667-4217b7f9-b5cc-428f-90ea-b3a8be4408cf.mov

**Review by jzhzhang66 — APPROVED** [review]
LGTMMMMM!!!! GO OFF KING

**Review by mike10911 — APPROVED** [review]
BRILLIANT LGTM

**Pull request merged**